WebUI/routes.js

```diff
@@ -6,8 +6,9 @@ router.get("/", function(req, res, next) {
   res.render("index", { title: "Yaydoc" });
 });
 
+/* Downloading Documentation as ZIP */
 router.get('/download/:email/:uniqueId', function (req, res, next) {
-  var file = __dirname + '/../temp/' + req.params.email + '/' + req.params.uniqueId + '.zip';
+  var file = __dirname + '/temp/' + req.params.email + '/' + req.params.uniqueId + '.zip';
   res.download(file);
 });
 
```

Comment:
Why do want to move this route to the main directory? since we going to build an integration platform in future. if we keep all the routes in the main directory in future. it'll be difficult to maintain if routes started to grow